src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -66,6 +66,7 @@ public async Task BulkDocs()
                     BatchTrafficWatch(command.ParsedCommands);
                 }
 
+                var disableAtomicDocumentWrites = GetBoolValueQueryString("disableAtomicDocumentWrites", required: false) ?? false;
```

Comment:
How does the user set the `disableAtomicDocumentWrites` option?
to maintain compatibility, maybe we need to allow setting this globally on the `DocumentStore`?

Reply:
Currently, they don't, haven't gotten to it

Reply:
And yes, need to enable that per session & globally.